test.txt

```diff
@@ -0,0 +1,2 @@
+test file new for git
```

Comment:
What is the reason for this file? Does not seem to be referenced in the tests 🤔

Reply:
For some reason git seems to pick it up sometimes from my local master branch, will investigate. Reason was to test git setup, but the file shouldn't have been pushed to this repo